utils/bmx7-dnsupdate/files/etc/uci-defaults/99bmx7-dnsupdate

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+/etc/init.d/bmx7-dnsupdate enable
+/etc/init.d/bmx7-dnsupdate start
```

Comment:
this is already handled in post-install by default, no need to handle enabling and starting the services yourself. please remove the uci-defaults script.

(did you see this anywhere in the repo? if so, we should drop it there as well.)

Reply:
I guess I adapted it from some LibreMesh packages :roll_eyes: